staging/src/k8s.io/apimachinery/pkg/watch/watch.go

```diff
@@ -27,7 +27,7 @@ import (
 
 // Interface can be implemented by anything that knows how to watch and report changes.
 type Interface interface {
-	// Stop stops watching. Will close the channel returned by ResultChan(). Releases
+	// Stop watching. Will close the channel returned by ResultChan(). Releases
```

Comment:
I think this wording was actually intended and correct. According to [godoc](https://tip.golang.org/doc/comment):

> As with packages (above) and funcs (below), doc comments for types start with complete sentences naming the declared symbol. An explicit subject often makes the wording clearer, and it makes the text easier to search, whether on a web page or a command line. For example:

See specifically the example for `Sort` showing a similar pattern:

```go
package sort

// Sort sorts data in ascending order as determined by the Less method.
// It makes one call to data.Len to determine n and O(n*log(n)) calls to
// data.Less and data.Swap. The sort is not guaranteed to be stable.
func Sort(data Interface) {
    ...
}
```